### DISCUSSION TITLE
Mixing Custom and Built-in Models

### DISCUSSION THREAD

**Discussion by asktimfisher** [dscripka/openWakeWord#93] [General]
Is there a way to mix custom and built in models? Specifically, I'd like to load 2 custom models and 1 built-in one. I know how to do either all built-in or one+ custom but unsure how to pull this off.

**Comment by dscripka** [comment]
This is certainly possible, though I'll admit it is a bit intuitive. You can load as many models as you like, including both custom and built-in models as long as you provide the full model paths. For example:

```
from openwakeword.model import Model

# Instantiate the model(s)
model = Model(
    wakeword_models = ["/path/to/my_custom_model1.tflite", "/path/to/my_custom_model2.tflite", "/path/to/builtin/model.tflite"]
)
```

For the built-in models, you can either find the location of the model inside of your local Python packages location, or just download them manually from [here](https://github.com/dscripka/openWakeWord/releases/tag/v0.5.1) so they are easier to work with.

**Reply by asktimfisher** [reply]
Great! Thanks so much!
